src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -143,4 +144,14 @@ public function getCondition()
     {
         return $this->condition;
     }
+
+    public function setUtf8($utf8)
+    {
+        $this->utf8 = $utf8;
+    }
+
+    public function isUtf8()
```

Comment:
do we need a new property on the route ? I suggested using options, which are already meant to provide hints to the route compiler